modules/ocf_printhost/manifests/enforcer.pp

```diff
@@ -1,6 +1,8 @@
 class ocf_printhost::enforcer {
   package { ['cups-tea4cups', 'mariadb-client']: }
 
+  $redis_password = assert_type(Pattern[/^[a-zA-Z0-9]*$/], hiera('broker::redis::password'))
```

Comment:
I'd take this opportunity to move enforcer.conf out of the private share and into the regular puppet repo as a template, like we do with ocfweb.conf or the broker.conf.erb in this PR